src/null-safety/index.md

```diff
@@ -61,33 +74,15 @@ Dart null safety support is based on the following three core design principles:
   fewer bugs, but smaller binaries and faster execution.
 
 
-## Enabling null safety {#enable-null-safety}
-
-Sound null safety is available in Dart 2.12 and Flutter 2.
-
+## Enabling/disabling null safety {#enable-null-safety}
 
-### Migrating an existing package or app {#migrate}
-
-For instructions on how to migrate your code to null safety,
-see the [migration guide][].
+Sound null safety is available in Dart 2.12 or later,
+and in Flutter 2.0 and later. Dart 3 and later will
+[_only_ support sound null safety][Dart 3 sound null safety tracking issue].
```

Comment:
Lots of "and later"s, maybe we can combine the first two:

```suggestion
Sound null safety is available
in Dart 2.12 and Flutter 2.0 or later,
Dart 3 and later will
[_only_ support sound null safety][Dart 3 sound null safety tracking issue].
```

Reply:
Hmm, I think "will require sound null safety" might be confused with "you can use null safety or not, but if you use it, it must be sound".  I still think "will only support sound null safety" reads stronger?